opentracing-apache-httpclient/src/main/java/io/opentracing/contrib/apache/http/client/TracingClientExec.java

```diff
@@ -71,16 +71,16 @@ public CloseableHttpResponse execute(
       HttpClientContext clientContext,
       HttpExecutionAware execAware) throws IOException, HttpException {
 
-    ActiveSpan localSpan = clientContext.getAttribute(ACTIVE_SPAN, ActiveSpan.class);
+    Scope localScope = clientContext.getAttribute(ACTIVE_SPAN, Scope.class);
```

Comment:
Just want to check  - is this called in the same thread that created the `Scope`? If so, then it could just use the `tracer.scopeManager().active()`?

Reply:
And if not, it would be worth a comment on the code addressing this :)

Reply:
I will check

Reply:
it fails

Reply:
Ok, so I think the ACTIVE_SPAN attribute would need to pass the `Span` rather than the `Scope` - as the scope is supposed to only be used within the thread.